web/src/app/alerts/components/AlertDetails.js

```diff
@@ -134,7 +238,10 @@ export default class AlertDetails extends Component {
           className={this.props.classes.tableCardContent}
           style={{ paddingBottom: 0 }}
         >
-          <List>{this.renderAlertLogEvents()}</List>
+          <AlertLogs
+            data={this.props.data.number}
```

Comment:
Kind of on @Auchindoun's point, we can move this to its own component that just takes an `alertID` and build it in hooks with the newer stuff in a separate file.